src/Layout/toolset-tasks/OverrideAndCreateBundledNETCoreAppPackageVersion.cs

```diff
@@ -122,11 +122,11 @@ void CheckAndReplaceAttribute(XAttribute attribute)
         /// so there is no need to replace them.
         /// </summary>
         /// <returns></returns>
-        private static bool IsSDKServicing(string sdkVersion)
+        private static bool IsNETServicing(string sdkVersion)
         {
             var parsedSdkVersion = NuGet.Versioning.NuGetVersion.Parse(sdkVersion);
```

Comment:
We want this to be tracking the runtime prerelease state rather than the SDK. So I'd change this from sdkVersion to netVersion and pass it originalBundledNETCoreAppPackageVersion (you'll have to reorder the code slightly above)